### PR TITLE
Add stories index

### PR DESCRIPTION
**Pull request by kenyonj**

### CODE REVIEW

File: db/migrate/20140603185502_add_active_to_stories.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddActiveToStories < ActiveRecord::Migration
+  def change
+    add_column :stories, :active, :boolean, default: false
```
Comment (mrgilman) [inline db/migrate/20140603185502_add_active_to_stories.rb:3]: You might want to add `null: false` to this, check out this blog post: http://robots.thoughtbot.com/avoid-the-threestate-boolean-problem

Comment (kenyonj) [inline db/migrate/20140603185502_add_active_to_stories.rb:3]: good call... I sometimes think that having the `default` set takes care of that... but I guess not.
thanks!

File: spec/features/user_views_stories_index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+
+feature "User views stories index" do
+  scenario "and sees only the active stories in a ul" do
+    user = create(:user)
+    story_1 = create(
+              :story,
```
Comment (mrgilman) [inline spec/features/user_views_stories_index_spec.rb]: I think this should just be indented two spaces past line 6.

**Comment by kenyonj** [comment]
thanks @mrgilman ... fixed those issues... everything else look good?

File: spec/features/user_views_stories_index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+
+feature "User views stories index" do
+  scenario "and sees only the active stories in a ul" do
+    user = create(:user)
+    story_1 = create(
```
Comment (mrgilman) [inline spec/features/user_views_stories_index_spec.rb:6]: Someone who knows more about thoughtbot's style guide might know better than me, but I think this should look like:

``` ruby
story_1 = create(
  :story,
  user: user,
  title: "Story Title 1",
  active: true
)
```

**Comment by mrgilman** [comment]
Those were the only things that stood out to me!

File: spec/features/user_views_stories_index_spec.rb (hunk 3 above)
Comment (kenyonj) [inline spec/features/user_views_stories_index_spec.rb:6]: Thank you @mrgilman ... You're right, confirmed by Pat.

**Pull request closed without merging**